modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/ViewConflictsMVCRenderCommand.java

```diff
@@ -75,6 +76,13 @@ public String render(
 			CTCollection ctCollection =
 				_ctCollectionLocalService.getCTCollection(ctCollectionId);
 
+			if ((ctCollection == null) ||
```

Comment:
We don't need the null check here because ctCollection will never be null. `get` throws a PortalException when the entity could not be found. `fetch` returns null.